modules/core/src/test/java/org/apache/ignite/internal/util/IgniteUtilsSelfTest.java

```diff
@@ -933,6 +933,70 @@ public void testCeilPow2() throws Exception {
             assertEquals(0, U.ceilPow2(i));
     }
 
+    /**
+     *
+     */
+    @Test
+    public void testCeilPow2Long() throws Exception {
+        assertEquals(2, U.ceilPow2Long(2));
+        assertEquals(4, U.ceilPow2Long(3));
+        assertEquals(4, U.ceilPow2Long(4));
+        assertEquals(8, U.ceilPow2Long(5));
+        assertEquals(8, U.ceilPow2Long(6));
+        assertEquals(8, U.ceilPow2Long(7));
+        assertEquals(8, U.ceilPow2Long(8));
+        assertEquals(16, U.ceilPow2Long(9));
+        assertEquals(1 << 15, U.ceilPow2Long((1 << 15) - 1));
+        assertEquals(1 << 15, U.ceilPow2Long(1 << 15));
+        assertEquals(1 << 16, U.ceilPow2Long((1 << 15) + 1));
+        assertEquals(1 << 26, U.ceilPow2Long((1 << 26) - 100));
+        assertEquals(1 << 26, U.ceilPow2Long(1 << 26));
+        assertEquals(1 << 27, U.ceilPow2Long((1 << 26) + 100));
+
+        assertEquals(1L << 61, U.ceilPow2Long((1L << 61) - 100));
+        assertEquals(1L << 61, U.ceilPow2Long(1L << 61));
+        assertEquals(1L << 62, U.ceilPow2Long((1L << 61) + 100));
+
+        assertEquals(1L << 62, U.ceilPow2Long((1L << 62) - 1));
+        assertEquals(1L << 62, U.ceilPow2Long(1L << 62));
+
+        try {
```

Comment:
Is it possible to replace this with assertThrows() ?

Reply:
Done